frontend/src/pages/Admin/Admin.tsx

```diff
@@ -1,13 +1,76 @@
-import { useContext } from "react"
+import { useContext, useEffect, useState } from "react"
 import { LoginContext } from "../../Components/LoginManager/LoginManager"
+import { approveSighting, fetchUnapprovedSightings, Sightings } from "../../api/backendClient"
+import "./Admin.scss"
 
 const Admin = () => {
-  const { roleType } = useContext(LoginContext)
+  const loginContext = useContext(LoginContext)
+  const [sightings, setSightings] = useState<Sightings | null>(null)
+  const [approvals, setApprovals] = useState(0)
 
-  if (roleType === "Admin") {
+  useEffect(() => {
+    fetchUnapprovedSightings(loginContext.jwt).then((response) => setSightings(response))
+  }, [loginContext.jwt, approvals])
```

Comment:
Why is `loginContext.jwt` in the useeffect? are we expecting it to be updated?

Reply:
It won't pass the pre-commit checks without having that in there

Reply:
strange but fair enough